--- irrlicht-1.5/source/Irrlicht/COpenGLExtensionHandler.cpp~	2008-12-14 07:15:56.000000000 +0100
+++ irrlicht-1.5/source/Irrlicht/COpenGLExtensionHandler.cpp	2008-12-16 08:57:41.000000000 +0100
@@ -198,7 +198,7 @@
 
 	#ifndef _IRR_GETPROCADDRESS_WORKAROUND_
 	__GLXextFuncPtr (*IRR_OGL_LOAD_EXTENSION)(const GLubyte*)=0;
-	#ifdef GLX_VERSION_1_4
+/*	#ifdef GLX_VERSION_1_4
 		int major,minor;
 		glXQueryVersion(glXGetCurrentDisplay(), &major, &minor);
 		if ((major>1) || (minor>3))
@@ -206,7 +206,7 @@
 		else
 	#endif
 			IRR_OGL_LOAD_EXTENSION=glXGetProcAddressARB;
-	#else
+	#else*/
 		#define IRR_OGL_LOAD_EXTENSION glXGetProcAddressARB
 	#endif
 	#endif
