--- irrlicht-1.3.1/source/Irrlicht/COpenGLExtensionHandler.cpp~	2007-06-07 15:17:34.000000000 +0200
+++ irrlicht-1.3.1/source/Irrlicht/COpenGLExtensionHandler.cpp	2007-07-20 21:34:19.000000000 +0200
@@ -188,7 +188,7 @@
 
 	#ifndef _IRR_GETPROCADDRESS_WORKAROUND_
 	__GLXextFuncPtr (*IRR_OGL_LOAD_EXTENSION)(const GLubyte*)=0;
-	#ifdef GLX_VERSION_1_4
+/*	#ifdef GLX_VERSION_1_4
 		int major,minor;
 		glXQueryVersion(glXGetCurrentDisplay(), &major, &minor);
 		if ((major>1) || (minor>3))
@@ -196,7 +196,7 @@
 		else
 	#endif
 			IRR_OGL_LOAD_EXTENSION=glXGetProcAddressARB;
-	#else
+	#else*/
 		#define IRR_OGL_LOAD_EXTENSION glXGetProcAddressARB
 	#endif
 	#endif
