--- irrlicht-1.8/source/Irrlicht/COpenGLExtensionHandler.cpp.orig	2013-07-14 08:59:10.710771070 +0200
+++ irrlicht-1.8/source/Irrlicht/COpenGLExtensionHandler.cpp	2013-07-14 09:20:05.454051748 +0200
@@ -389,7 +389,7 @@
 	// next line by uncommenting it
 	// #define _IRR_GETPROCADDRESS_WORKAROUND_
 
-	#ifndef _IRR_GETPROCADDRESS_WORKAROUND_
+	#if 0
 	__GLXextFuncPtr (*IRR_OGL_LOAD_EXTENSION_FUNCP)(const GLubyte*)=0;
 	#ifdef GLX_VERSION_1_4
 		int major=0,minor=0;
