--- irrlicht-1.4beta/source/Irrlicht/COpenGLExtensionHandler.cpp~	2007-10-04 19:17:51.000000000 +0200
+++ irrlicht-1.4beta/source/Irrlicht/COpenGLExtensionHandler.cpp	2007-10-04 19:19:24.000000000 +0200
@@ -179,7 +179,7 @@
 
 	#ifndef _IRR_GETPROCADDRESS_WORKAROUND_
 	__GLXextFuncPtr (*IRR_OGL_LOAD_EXTENSION)(const GLubyte*)=0;
-	#ifdef GLX_VERSION_1_4
+/*	#ifdef GLX_VERSION_1_4
 		int major,minor;
 		glXQueryVersion(glXGetCurrentDisplay(), &major, &minor);
 		if ((major>1) || (minor>3))
@@ -187,7 +187,7 @@
 		else
 	#endif
 			IRR_OGL_LOAD_EXTENSION=glXGetProcAddressARB;
-	#else
+	#else*/
 		#define IRR_OGL_LOAD_EXTENSION glXGetProcAddressARB
 	#endif
 	#endif
